src/Nri/Ui/AssignmentIcon/V1.elm

```diff
@@ -277,3 +278,232 @@ startSecondary =
+                        , Svg.path [ Attributes.d "M18.8324383,37.3886648 C18.5195828,37.0758093 18.5195828,36.6127831 18.8324383,36.2999276 C19.1452938,35.9870721 19.60832,35.9870721 19.9211755,36.2999276 L21.3227682,37.7015203 L22.724361,36.2999276 C23.0372165,35.9870721 23.5565566,35.9870721 23.863155,36.2999276 C24.1760106,36.6127831 24.1760106,37.1321233 23.863155,37.4387217 L22.4615623,38.8403144 L23.863155,40.2419071 C24.1760106,40.5547626 24.1760106,41.0177888 23.863155,41.3306443 C23.5502995,41.6434998 23.0872734,41.6434998 22.7744178,41.3306443 L21.3728251,39.9290516 L19.9712324,41.3306443 C19.6583769,41.6434998 19.1390367,41.6434998 18.8324383,41.3306443 C18.5195828,41.0177888 18.5195828,40.4984486 18.8324383,40.1918502 L20.234031,38.8403144 L18.8324383,37.3886648 Z", Attributes.fill "#FFFFFF" ] []
+                        , Svg.path [ Attributes.d "M27.7066041,32.1644352 C26.993092,32.1644352 26.4038432,31.6179437 26.3383711,30.9191294 L26.3397072,30.7935298 C26.3397072,30.0359242 26.9543432,29.418616 27.7092764,29.418616 L50.2744322,29.418616 C51.2872454,29.418616 51.6466738,30.1588514 51.6466738,30.7935298 C51.6466738,31.4255359 51.2872454,32.1644352 50.2744322,32.1644352 L27.7066041,32.1644352 Z", Attributes.fill "#FFFFFF" ] []
+                        , Svg.path [ Attributes.d "M27.7066041,48.2119559 C26.9516708,48.2119559 26.3343626,47.5959838 26.3330264,46.8370421 C26.3330264,46.083445 26.9476623,45.4661367 27.7025956,45.4661367 L50.2744322,45.4661367 C51.2872454,45.4661367 51.6466738,46.2063721 51.6466738,46.8410506 C51.6466738,47.4730566 51.2872454,48.2119559 50.2744322,48.2119559 L27.7066041,48.2119559 Z", Attributes.fill "#FFFFFF" ] []
+                        , Svg.path [ Attributes.d "M27.7066041,40.1881956 C26.9516708,40.1881956 26.3343626,39.5708873 26.3330264,38.8132817 C26.3330264,38.0583484 26.9476623,37.4423763 27.7025956,37.4423763 L50.2744322,37.4423763 C51.2872454,37.4423763 51.6466738,38.1812756 51.6466738,38.815954 C51.6466738,39.4479601 51.2872454,40.1881956 50.2744322,40.1881956 L27.7066041,40.1881956 Z", Attributes.fill "#FFFFFF" ] []
+                        ]
+                    ]
+                ]
+            ]
+        ]
+        |> Nri.Ui.Svg.V1.fromHtml
+
+
+{-| -}
+unitDiagnostic : Nri.Ui.Svg.V1.Svg
+unitDiagnostic =
+    Svg.svg
+        [ Attributes.width "100%"
+        , Attributes.height "100%"
+        , Attributes.viewBox "0 0 71 71"
+        , Attributes.fill "currentcolor"
+        ]
+        [ Svg.g [ Attributes.fillRule "evenodd" ]
+            [ Svg.g [ Attributes.transform "translate(0.150183, 0.000000)" ]
```

Comment:
I bet we can remove this translate -- seems like a teeny tiny adjustment that applies to all the svg children in this icon